### PR TITLE
ROX-16625: Enable OCP 4.13 for ACS 3.73

### PR DESCRIPTION
**Pull request by BradLugo**
/assign @BradLugo
/cc @gavin-stackrox @msugakov
/pj-rehearse auto-ack

### CODE REVIEW

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.73__ocp-4-13.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 5h0m0s
+- as: merge-qa-e2e-tests
+  postsubmit: true
+  steps:
+    cluster_profile: aws
+    env:
+      QA_TEST_TARGET: test
```
Comment (msugakov) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.73__ocp-4-13.yaml:51]: Similarly to https://github.com/openshift/release/pull/38678#discussion_r1179631178 what's the role of `QA_TEST_TARGET: test`?

**Review by msugakov — APPROVED** [review]
/lgtm
/hold

**Comment by BradLugo** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-3.73-ocp-4-13-qa-e2e-tests

**Comment by BradLugo** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-release-3.73-ocp-4-13-qa-e2e-tests

**Comment by BradLugo** [comment]
Same deal from https://github.com/openshift/release/pull/38678#issuecomment-1535058162

/pj-rehearse ack

**Pull request merged**